Source/SwipeActionsView.swift

```diff
@@ -17,7 +17,7 @@ class SwipeActionsView: UIView {
     let transitionLayout: SwipeTransitionLayout
     var layoutContext: ActionsViewLayoutContext
     
-    var expansionAnimator: UIViewPropertyAnimator?
+    var expansionAnimator: Any?
```

Comment:
Need to reuse the `SwipeAnimator` here. See my comments below regarding matching the `SwipeAnimator` protocol to the `UIViewPropertyAnimator` class for the initializer and methods that we use.